modules/tinymce/src/plugins/table/main/ts/ui/TableDialog.ts

```diff
@@ -108,7 +109,11 @@ const onSubmitTableForm = (editor: Editor, tableElm: HTMLTableElement | undefine
       const cols = parseInt(data.cols, 10) || 1;
       const rows = parseInt(data.rows, 10) || 1;
       // Cases 1 & 3 - inserting a table
-      tableElm = InsertTable.insert(editor, cols, rows, 0, 0);
+      editor.execCommand('mceInsertTable', false, { rows, columns: cols });
+      tableElm = TableSelection.getSelectionCell(Utils.getSelectionStart(editor), Utils.getIsRoot(editor))
+        .bind((cell) => TableLookup.table(cell, Utils.getIsRoot(editor)))
+        .map((table) => table.dom)
+        .getOrUndefined();
```

Comment:
I feel like this was an opportunity to move all the 'add these attributes to the table' things into the insert table event, instead of looking up the new table and patching it, but oh well. I'm not sure how we'd do that in a nice extensible way.